### ISSUE TITLE
[BUG] Project loading failing if $HOME directory path contains whitespaces

### ISSUE THREAD

**Issue by senketsu03** [microsoft/vscode-dotnettools#165]
### Describe the Issue

Opening a project directory on a windows pc produces project loading error message from extension. It doesn't matter if project's path doesn't contains whitespaces, judging by the logs it tries to find some `.NET program` in a non-existent directory. Log:

*Note: my $HOME directory path is `C:\Users\User Name`*

```sh
2023-06-19 10:24:25.739 [info] Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-c:\Users\User does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

2023-06-19 10:24:27.746 [info] Project system initialization finished. 0 project(s) are loaded, and 4 failed to load.
```

### Steps To Reproduce

Open a solution/project directory on a windows system, which user's name (and therefore `$HOME` path) contains whitespaces

### Expected Behavior

Projects are loaded

### Environment Information

- windows 11
- vscode 1.79.2
- extensions: c# 2.0.212, c# dev kit 0.1.103
- projects are using `net7.0` framework defined in `Directory.Build.props` file

**Comment by timheuer** [comment]
Looks like dupe of #24

**Comment by Michael-Eng** [comment]
@senketsu03 we've released an update to fix #24, please try that build to see if this issue is fixed for you? Note this is a prerelease build of C# Dev Kit.

CC: @Jxwoon

**Comment by senketsu03** [comment]
Yes, that did fix the issue, thank you 👍
